spec/factories.rb

```diff
@@ -1,10 +1,15 @@
 FactoryBot.define do
-  sequence :movie_title do |n|
-    "Movie Title #{n}"
+  factory :ingredient do
+    name { generate(:name) }
   end
 
-  sequence :name do |n|
-    "name #{n}"
+  factory :ingredient_mention do
```

Comment:
I still think that movie { generate(:movie_id)} should just be movie, and ingredient { generate(:ingredient_id) } should just be ingredient. Try it like that plz.